lib/Remote/HttpCommandExecutor.php

```diff
@@ -242,7 +262,7 @@ public function execute(WebDriverCommand $command)
             }
         }
 
-        if ($params && is_array($params) && $http_method !== 'POST') {
+        if ($params && \is_array($params) && $http_method !== 'POST') {
```

Comment:
I wouldn't change this as part of this PR (also on other lines).

The codestyle may be changed, but then we can do this at once in whole codebase, and do not change it randomly to keep this PR self-contained.

Reply:
I'll revert these changes (can't remember why I've done that)

Reply:
probably phpstorm suggestion :)